### ISSUE TITLE
H340 and i7-9700 issue

### ISSUE THREAD

**Issue by tabmag** [openhardwaremonitor/openhardwaremonitor#1145]
The i7-9700 only shows Load. Missing Temperatures, Bus Speed, and Powers.

**Comment by tabmag** [comment]
I  restarted as admin, it still only shows Load for the i7.. I don't know what you mean by add another one. I don't se add as an option anywhere.

**Comment by DarthDestroyer** [comment]
I meant the Report!
Without Admin, it will not work

**Comment by DarthDestroyer** [comment]
And what i can say instantly => http://openhardwaremonitor.org/files/openhardwaremonitor-v0.8.0.3-alpha.zip

**Comment by tabmag** [comment]
[OpenHardwareMonitor.Report.txt](https://github.com/openhardwaremonitor/openhardwaremonitor/files/2592236/OpenHardwareMonitor.Report.txt)

**Comment by DarthDestroyer** [comment]
Chip ID: Unknown Winbond / Nuvoton / Fintek with ID 0xD121 at 0x2E/0x2F

**Comment by tabmag** [comment]
The v0.8.0.3 alpha works.

**Comment by DarthDestroyer** [comment]
For the Mainboard => #981 should help, tested in #1060

**Comment by tabmag** [comment]
Yeah, the v0.8.0.3 alpha works for the i7 issue, but not the Mainboard.

**Comment by pkhdsth** [comment]
you can download https://www.cpuid.com/downloads/hwmonitor/hwmonitor_1.37.zip
run it, then upload report of hwmonitor, hope i can help you :D

**Comment by DarthDestroyer** [comment]
@pkhdsth it is already solved ;-)

**Comment by tabmag** [comment]
The CPUID HWMonitor displays the mother board information, Open Hardware Manager does not, but CPUID HWMonitor has no option to change the temperature to Farenheit or to run at startup. And Open Hardware Manager loads much faster, if only it would display the motherboard info.

**Comment by tabmag** [comment]
Oh, CPUID does show Farenheit in parenthesis next to Celsius.
